### PR TITLE
Replace whitelist strings references in English and Spanish

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/414730916066338/1180899597720226
Tech Design URL: 
CC: 

**Description**:
This PR replaces references to "whitelist" in English and Spanish. Until we get new translations for all the languages we have moved the English strings to the untranslated file for clarity.

**Steps to test this PR**:
1. Launch the app.
1. Go to a website.
1. Open the menu and see there are no references to whitelist.
1. Open settings and see there are no references to whitelist.
1. Open "Unprotected sites" and see there are no references to whitelist.
1. Repeat in Spanish with no references to "lista blanca".

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by marcosholgado** [comment]
@cmonfortep, @malmstein I'm adding you both here to review this PR because of the Spanish translations in case you can think of something better.

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -415,27 +415,26 @@
...
-    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio web de la lista blanca %s</string>
-    <string name="whitelistEntryDeleteConfirmMessage">¿Estás seguro de que desea eliminar &lt;b&gt;%s&lt;/b&gt; de la lista blanca?</string>
-    <string name="whitelistExplanation">Estos sitios web incluidos en la lista blanca no se actualizarán por Protección de Privacidad</string>
-    <string name="whitelistNoEntries">Aún no hay sitios web en la lista blanca</string>
+    <string name="whitelisetActivityTitle">Gestionar Protección</string>
+    <string name="whitelistAdd">Habilitar Protección</string>
+    <string name="whitelistRemove">Deshabilitar Protección</string>
+    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio desprotegido %s</string>
+    <string name="whitelistEntryDeleteConfirmMessage">Seguro que quieres habilitar de nuevo Protección de Privacidad para el sitio &lt;b>%s&lt;/b>?</string>
+    <string name="whitelistExplanation">Estos sitios no estarán mejorados por Protección de Privacidad</string>
```
Comment (malmstein) [inline app/src/main/res/values-es/strings.xml:431]: Does `enhanced` translate to `mejorados`? It reads weird, maybe something like "La Protección de Privacidad no esta aplicada a estos sitios''?

Comment (marcosholgado) [reply app/src/main/res/values-es/strings.xml:431]: Mi gusta

Comment (marcosholgado) [reply app/src/main/res/values-es/strings.xml:431]: Enhanced translates to "mejorar" but it does sound a bit weird. How about... "Estos sitios no utilizarán la Protección de Privacidad"? Saying "aplicada" sounds like applied which is weird to me but I'm not too strong about one or the other.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -415,27 +415,26 @@
...
+    <string name="whitelistAdd">Habilitar Protección</string>
+    <string name="whitelistRemove">Deshabilitar Protección</string>
+    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio desprotegido %s</string>
+    <string name="whitelistEntryDeleteConfirmMessage">Seguro que quieres habilitar de nuevo Protección de Privacidad para el sitio &lt;b>%s&lt;/b>?</string>
+    <string name="whitelistExplanation">La Protección de Privacidad no está aplicada a estos sitios</string>
+    <string name="whitelistNoEntries">Protección de Privacidad está habilitado para todos los sitios.</string>
     <string name="whitelistDomainHint">www.ejemplo.com</string>
-    <string name="whitelistFormatError">No se ha podido guardar, ingresa un dominio como <b>ejemplo.com</b>
-  </string>
+    <string name="whitelistFormatError">No se ha podido guardar, ingresa un dominio como <b>ejemplo.com</b></string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:434]: `introduce` better than`ingresa`.

What about? `Ha habido un error al guardar, introduce un dominio válido como ejemplo.com`

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -415,27 +415,26 @@
...
-    <string name="whitelistEntryDeleteConfirmMessage">¿Estás seguro de que desea eliminar &lt;b&gt;%s&lt;/b&gt; de la lista blanca?</string>
-    <string name="whitelistExplanation">Estos sitios web incluidos en la lista blanca no se actualizarán por Protección de Privacidad</string>
-    <string name="whitelistNoEntries">Aún no hay sitios web en la lista blanca</string>
+    <string name="whitelisetActivityTitle">Gestionar Protección</string>
+    <string name="whitelistAdd">Habilitar Protección</string>
+    <string name="whitelistRemove">Deshabilitar Protección</string>
+    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio desprotegido %s</string>
+    <string name="whitelistEntryDeleteConfirmMessage">Seguro que quieres habilitar de nuevo Protección de Privacidad para el sitio &lt;b>%s&lt;/b>?</string>
+    <string name="whitelistExplanation">La Protección de Privacidad no está aplicada a estos sitios</string>
+    <string name="whitelistNoEntries">Protección de Privacidad está habilitado para todos los sitios.</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:432]: `Protección de Privacidad habilitada para todos los sitios.`

File: app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -415,27 +415,26 @@
...
-    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio web de la lista blanca %s</string>
-    <string name="whitelistEntryDeleteConfirmMessage">¿Estás seguro de que desea eliminar &lt;b&gt;%s&lt;/b&gt; de la lista blanca?</string>
-    <string name="whitelistExplanation">Estos sitios web incluidos en la lista blanca no se actualizarán por Protección de Privacidad</string>
-    <string name="whitelistNoEntries">Aún no hay sitios web en la lista blanca</string>
+    <string name="whitelisetActivityTitle">Gestionar Protección</string>
+    <string name="whitelistAdd">Habilitar Protección</string>
+    <string name="whitelistRemove">Deshabilitar Protección</string>
+    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio desprotegido %s</string>
+    <string name="whitelistEntryDeleteConfirmMessage">Seguro que quieres habilitar de nuevo Protección de Privacidad para el sitio &lt;b>%s&lt;/b>?</string>
+    <string name="whitelistExplanation">La Protección de Privacidad no está aplicada a estos sitios</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:431]: I would replace `aplicacada` for `habilitada`, or in this case to use `deshabilitada`, similar but just to keep consistency with other translations. 

`La Protección de Privacidad está deshabilitada para estos sitios`

File: app/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -415,27 +415,26 @@
...
-    <string name="whitelistAdd">Agregar a la lista blanca</string>
-    <string name="whitelistRemove">Eliminar de la lista blanca</string>
-    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio web de la lista blanca %s</string>
-    <string name="whitelistEntryDeleteConfirmMessage">¿Estás seguro de que desea eliminar &lt;b&gt;%s&lt;/b&gt; de la lista blanca?</string>
-    <string name="whitelistExplanation">Estos sitios web incluidos en la lista blanca no se actualizarán por Protección de Privacidad</string>
-    <string name="whitelistNoEntries">Aún no hay sitios web en la lista blanca</string>
+    <string name="whitelisetActivityTitle">Gestionar Protección</string>
+    <string name="whitelistAdd">Habilitar Protección</string>
+    <string name="whitelistRemove">Deshabilitar Protección</string>
+    <string name="whitelistEntryOverflowContentDescription">Más opciones para el sitio desprotegido %s</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:429]: it sounds better `no protegido` than `desprotegido`, but no strong opinion.

File: app/src/main/res/values-es/strings.xml [hunk 6]
```diff
@@ -415,27 +415,26 @@
...
+    <string name="whitelistExplanation">La Protección de Privacidad no está aplicada a estos sitios</string>
+    <string name="whitelistNoEntries">Protección de Privacidad está habilitado para todos los sitios.</string>
     <string name="whitelistDomainHint">www.ejemplo.com</string>
-    <string name="whitelistFormatError">No se ha podido guardar, ingresa un dominio como <b>ejemplo.com</b>
-  </string>
+    <string name="whitelistFormatError">No se ha podido guardar, ingresa un dominio como <b>ejemplo.com</b></string>
 
     <!-- Settings -->
-    <string name="settingsPrivacyProtectionWhitelist">Lista blanca de Protección de Privacidad</string>
+    <string name="settingsPrivacyProtectionWhitelist">Sitios desprotegidos</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:437]: The same than in a previous comment. it sounds better to me `Sitios no protegidos` than `desprotegidos`.

**Comment by marcosholgado** [comment]
All changes done! Thanks a lot @malmstein and @cmonfortep for the feedback! I'll merge once the product review is approved.

**Pull request merged**